src/components/Filter/utils/pipelineruns-filter-utils.ts

```diff
@@ -12,17 +12,19 @@ export const filterPipelineRuns = (
   pipelineRuns: PipelineRunKind[],
   filters: PipelineRunsFilterState,
   customFilter?: (plr: PipelineRunKind) => boolean,
+  componentName?: string,
 ): PipelineRunKind[] => {
   const { name, status, type } = filters;
 
   return pipelineRuns
     .filter((plr) => {
       const runType = plr?.metadata.labels[PipelineRunLabel.PIPELINE_TYPE];
       return (
-        (!name ||
-          plr.metadata.name.indexOf(name) >= 0 ||
-          plr.metadata.labels?.[PipelineRunLabel.COMPONENT]?.indexOf(name.trim().toLowerCase()) >=
-            0) &&
+        (!name || plr.metadata.name.indexOf(name) >= 0) &&
+        (!componentName ||
```

Comment:
as I explained in the commit message, the reason of this change is the following:

```
Update the filter logic to filter by "component name" separately from the PLR name. 
This change makes the filtering behavior consistent with how PLRs are filtered from tekton-results.
```